popde/utils_plot.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+def simple2Dplot(xx, yy, kde, xlabel=None, ylabel=None, title=None):
+    """
+    return kde plot in 2D given output
+    """
+    fig = plt.figure()
+    ax = fig.gca()
+    # Filled Contour plot
+    cfset = ax.contourf(xx, yy, kdevals, cmap='Blues')
+    # Contour plot
+    cset = ax.contour(xx, yy, kdevals, colors='k')
+    # Contour line labels
+    ax.clabel(cset, inline=1, fontsize=10)
+    if xlabel:
+        ax.set_xlabel(xlabel)
+    if ylabel:
+        ax.set_ylabel(ylabel)
+    if title:
+        ax.set_title(title)
+    return fig
+
+def get_twoD_grid(data_dim1, data_dim2, num_points=100):
```

Comment:
It would be nice to have a 'pad' option which puts a slightly bigger range of the coordinate so that we don't get points exactly on the boundaries.  If we define dim1_range as dim1_max - dim1_min,  eg 'pad=0.01' would take the linspace to be between (dim1_min - 0.01 * dim1_range) and (dim1_min + 0.01 * dim1_range) and the same for dim2. 

Also it would be nice to have dim1 range and dim2 range options so that if wanted the user could specify whatever they wanted.

Reply:
added